### DISCUSSION TITLE
how to enable feature "showing directory on top of terminal" ?

### DISCUSSION THREAD

**Discussion by ghost** [starship/starship#2796] [Q&A]
I am using kitty terminal and fish shell.
I want my terminal look like the following one.

how can I enable starship shell prompt **to show directory** and **the shell I am using** at the top of terminal?

via this https://starship.rs/advanced-config/#change-window-title ?
i couldn't find the proper settings for fish shell

<img
  src="https://raw.githubusercontent.com/starship/starship/master/media/demo.gif"
  alt="Starship with iTerm2 and the Snazzy theme"
  width="80%"
/>

**Comment by fitrh** [comment]
Create a function under `$XDG_CONFIG_HOME/fish/functions` called `fish_title.fish` and put this

```fish
function fish_title
    printf "%s %s" (basename $SHELL) (pwd)
end
```
